### DISCUSSION TITLE
ReadMe needs to be updated

### DISCUSSION THREAD

**Discussion by ShreyasTitus** [divyagiridhar/SE_Project_2#15] [General]
The readme file has to be updated based on the new features which we discussed and added

**Comment by divyagiridhar** [comment]
Preparing the video

**Comment by divyagiridhar** [comment]
Here is the zenodo link - 
<a href="https://zenodo.org/badge/latestdoi/573616606"><img src="https://zenodo.org/badge/573616606.svg" alt="DOI"></a>
